docs/api/previous-rancher-api-versions/previous-rancher-api-versions.md

```diff
@@ -1,14 +1,14 @@
 ---
-title: API
+title: Previous API Guide
 ---
 
 <head>
-  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/reference-guides/about-the-api"/>
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/api/previous-rancher-api-versions"/>
 </head>
 
-## How to use the API
+## How to Use the API
 
-The API has its own user interface accessible from a web browser.  This is an easy way to see resources, perform actions, and see the equivalent cURL or HTTP request & response.  To access it:
+The past Rancher versions `v2.7.x` and below have their own API user interfaces accessible from a [web browser](https://ranchermanager.docs.rancher.com/v2.7/reference-guides/about-the-api). This is an easy way to see resources, perform actions, and see the equivalent cURL or HTTP request & response.  To access it:
```

Comment:
This feature has not been removed yet and is still available for the 2.8 line. 

In the case where it is removed, linking to `v2.7/reference-guides/about-the-api` doesn't really make sense since there's no content difference between the 2.7 and 2.8 versions of the page, so it's pretty much a self-referencing link.